src/index.js

```diff
@@ -520,15 +519,23 @@ function createConfig(options, entry, format, writeMeta) {
 							],
 						],
 					}),
-					// Only used for async await
 					babel({
-						// We mainly use bublé to transpile JS and only use babel to
-						// transpile down `async/await`. To prevent conflicts with user
-						// supplied configurations we set this option to false. Note
-						// that we never supported using custom babel configs anyway.
-						babelrc: false,
 						extensions: EXTENSIONS,
 						exclude: 'node_modules/**',
+						presets: [
+							[
+								'@babel/preset-env',
+								{
+									loose: true,
+									modules: false,
+									useBuiltIns: 'usage',
+									corejs: 2,
+									targets:
+										options.target === 'node' ? { node: '6' } : undefined,
+									exclude: ['transform-async-to-generator'],
```

Comment:
Are we transforming regenerator calls?

Reply:
We're transforming `async/await` via the excellent [babel-plugin-transform-async-to-promises](https://github.com/rpetrich/babel-plugin-transform-async-to-promises) 🎉

Reply:
Yes but the regenerator transform is not excluded here, isn’t that also in scope of that plugin

Reply:
I don't think it's needed as async await gets transpiled to promises which means no generators are used. So no generators polyfills are imported.